skimage/graph/_mcp.pyx

```diff
@@ -415,6 +419,8 @@ cdef class MCP:
             end-position will be found; otherwise the algorithm will stop when
             a a path is found to any end-position. (If no `ends` were
             specified, then this parameter has no effect.)
+        max_cost : float, optional
+            Threshold for the local costs above which an index is not considered.
```

Comment:
I think "local costs" is a term used or explained elsewhere. Might be a good idea to use different wording. Maybe

"Cost limit for each step between points. Points whose costs is higher than this will form a barrier."

Since this parameter doesn't refer to the cumulative cost used by the returned by `cumulative_costs`, can we think of a more fitting name? E.g., what do you think about `max_step_cost`?

Reply:
Applied the suggested wording for the parameter name and its description.